src/smartHandler.ts

```diff
@@ -59,20 +61,12 @@ export class SMARTHandler implements Authorization {
         }
 
         // Verify token
-        let response;
-        try {
-            response = await axios.post(
-                this.config.authZUserInfoUrl,
-                {},
-                { headers: { Authorization: `Bearer ${request.accessToken}` } },
-            );
-        } catch (e) {
-            console.error('Post to authZUserInfoUrl failed', e);
-        }
-        if (!response || !response.data[this.config.expectedFhirUserClaimKey]) {
-            console.error(`result from AuthZ did not have ${this.config.expectedFhirUserClaimKey} claim`);
-            throw new UnauthorizedError("Cannot determine requester's identity");
+        const results = [];
+        for (let i = 0; i < this.config.authStrategies.length; i += 1) {
+            const strategy = this.config.authStrategies[i];
+            results.push(strategy.isTokenValid(request.accessToken));
         }
+        await Promise.all(results);

```

Comment:
You should be able to streamline this like: 
```suggestion
        const results = this.config.authStrategies.map( strategy => { return strategy(accessToken) } );
        await Promise.all(results);
```

Reply:
Much cleaner! Thanks.